module7_selenium/src/test/java/ICanWinTestMain.java

```diff
@@ -0,0 +1,25 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import page.PastebinStartPage;
+
+public class ICanWinTestMain {
+    private WebDriver driver;
+
+    @BeforeMethod(alwaysRun = true)
+    public void browserSetup() {
+        driver = new ChromeDriver();
+    }
```

Comment:
add `driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS)` to configure timouts globally